### ISSUE TITLE
missing deb package dependency

### ISSUE THREAD

**Issue by dosaboy** [canonical/hotsos#709]
```
  # hotsos
  Traceback (most recent call last):
    File "/usr/bin/hotsos", line 11, in <module>
      load_entry_point('hotsos==1.16.0+1138.ubuntu20.4.1', 'console_scripts', 'hotsos')()
    File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 490, in load_entry_point
      return get_distribution(dist).load_entry_point(group, name)
    File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2854, in load_entry_point
      return ep.load()
    File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2445, in load
      return self.resolve()
    File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2451, in resolve
      module = __import__(self.module_name, fromlist=['__name__'], level=0)
    File "/usr/lib/python3/dist-packages/hotsos/cli.py", line 12, in <module>
      from hotsos.core.root_manager import DataRootManager
    File "/usr/lib/python3/dist-packages/hotsos/core/root_manager.py", line 9, in <module>
      from hotsos.core.host_helpers import CLIHelper
    File "/usr/lib/python3/dist-packages/hotsos/core/host_helpers/__init__.py", line 26, in <module>
      from .systemd import (  # noqa: F403,F401
    File "/usr/lib/python3/dist-packages/hotsos/core/host_helpers/systemd.py", line 7, in <module>
      import dateutil
  ModuleNotFoundError: No module named 'dateutil'
```

**Comment by dosaboy** [comment]
have added missing deps to debian/control so hopefully good now, waiting for builds to finish

**Comment by dosaboy** [comment]
fixed as of deb version 1.16.0+1139
